math/finding_number_of_Digits_in_a_Number.cpp

```diff
@@ -0,0 +1,42 @@
+/**
+ * @author [aminos 🇮🇳](https://github.com/amino19)
+ * @file
+ *
+ * @brief [Program to count digits 
+ * in an integer](https://www.geeksforgeeks.org/program-count-digits-integer-3-different-methods)
+ * @details It is a very basic math of finding number of digits in a given
+ * number i.e, we can use it by inputting values whether it can be a
+ * positive/negative value, let's say: an integer. There is also a second method: 
+ * by using "K = floor(log10(N) + 1)", but it's only applicable for
+ * numbers (not integers).
+ * For more details, refer to the [Algorithms-Explanation](https://github.com/TheAlgorithms/Algorithms-Explanation/blob/master/en/Basic%20Math/Finding the number of digits in a number.md) repository.
+ */
+
+#include <cassert>   /// for assert
+#include <iostream>  /// for IO operations
+
```

Comment:
Please add self-test implementations using `assert` (see the [typical structure](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#typical-structure-of-a-program) of a program as an example).

Reply:
@Panquesito7 : Help me for this! :smile: